spel/scripts/amigen7-build.sh

```diff
@@ -17,13 +17,14 @@ BUILDNAME="${SPEL_BUILDNAME}"
 CHROOT="${SPEL_CHROOT:-/mnt/ec2-root}"
 CLOUDPROVIDER="${SPEL_CLOUDPROVIDER:-aws}"
 CUSTOMREPORPM="${SPEL_CUSTOMREPORPM}"
-CUSTOMREPONAME="${SPEL_CUSTOMREPONAME}"
```

Comment:
CUSTOMREPONAME and CUSTOMREPORPM are related. CUSTOMREPORPM is the package that provides the repos listed in CUSTOMREPONAME. If we rename CUSTOMREPONAME, we should also rename CUSTOMREPORPM

Reply:
Also, these envs are back by packer variables, mapped through the packer template. If we change the envs, we need to update the template and the variable names.